### PR TITLE
add the required dependency and rewrite the tasks using YAML syntax

### PR DESCRIPTION
**Pull request by arbabnazar**
@edx/devops kindly review. Thanks

@mulby kindly review the change. It need some `COMMON` variables which are located in `common_vars` roles which added as dependency. 

Thanks,

### CODE REVIEW

File: playbooks/roles/hadoop_common/tasks/main.yml [hunk 1]
```diff
@@ -22,174 +22,202 @@
...
+    state: link
+
+- name: Configuration installed
+  template:
+    src: "{{ item }}.j2"
+    dest: "{{ HADOOP_COMMON_CONF_DIR }}/{{ item }}"
+    mode: "0640"
+    owner: "{{ hadoop_common_user }}"
+    group: "{{ hadoop_common_group }}"
   with_items:
```
Comment (e0d) [inline playbooks/roles/hadoop_common/tasks/main.yml:106]: Same comment about making these anonymous dicts with owner/group/mode, etc.  If you have time.

**Comment by e0d** [comment]
Small comment, but generally :+1:

**Comment by mulby** [comment]
:+1:

**Comment by arbabnazar** [comment]
@e0d please review. Thanks

**Comment by arbabnazar** [comment]
@e0d please review the changes and let me know if you want modification. Thanks

**Comment by arbabnazar** [comment]
@e0d please review. Thanks

**Comment by arbabnazar** [comment]
@e0d @mulby  can you please review and merge it. Thanks

**Pull request closed without merging**